network_simulator/host.py

```diff
@@ -23,4 +28,91 @@ def __init__(self, address=None, flows={}, flow_packets_received={},
         self.flow_packets_received = flow_packets_received
         self.link = link
 
-    # TODO(team): More event subclasses to come?
\ No newline at end of file
+
+class HostReceivedPacketEvent(Event):
+    """
+    Host receives packets from a flow.
+    """
+
+    def __init__(self, host, packet, packet_previously_received):
```

Comment:
You should not include packet_previously_received as an argument to the constructor.